app/javascript/components/pages/ShowPet.js

```diff
@@ -27,6 +27,21 @@ class ShowPet extends Component{
         }
     render(){
         let { pet } = this.props
+        let newMedicalText = pet.medical.split('\n').map((item, i) => {
```

Comment:
Would be nice to turn this into a function that allows you to run it on each block of text, something like this:
```suggestion
        const removeNewLineChar = (target) => {
            return pet[target].split('\n').map((item, i) => {
               return <p key={i}>{item}</p>;
            }
         }      
```